__tests__/app.test.js

```diff
@@ -203,3 +205,33 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("responds with an array of objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.users)).toBe(true);
+        body.users.forEach((item) => {
+          return expect(item).toBeInstanceOf(Object);
+        });
+      });
+  });
+  test("contains the properties: username, name, avatar_url", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        body.users.forEach((item) => {
```

Comment:
If you're going to use a for each in a test, you must assert that the array has a length of at least one. Something like this may help https://jestjs.io/docs/expect#toBeGreaterThan